packages/magento-cart/components/ApolloCartError/ApolloCartErrorAlert.tsx

```diff
@@ -19,13 +19,14 @@ export function ApolloCartErrorAlert(props: ApolloCartErrorAlertProps) {
     <ApolloCustomerErrorAlert
       {...props}
       graphqlErrorAlertProps={{
-        action: noSuchEntity ? (
-          <Button onClick={clear}>
-            <Trans id='Reset Cart' />
-          </Button>
-        ) : (
-          graphqlErrorAlertProps?.action
-        ),
+        action:
+          noSuchEntity && resetCart ? (
+            <Button onClick={clear} size='small'>
+              <Trans id='Reset Cart' />
+            </Button>
+          ) : (
+            graphqlErrorAlertProps?.action
+          ),
```

Comment:
Please revert, will be fixed in https://github.com/graphcommerce-org/graphcommerce/pull/1608